src/demo/app/fancy-form/fancy-field-demo.component.ts

```diff
@@ -0,0 +1,264 @@
+          <div class="fancy-hint">Give us your best guess.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>checkbox</code></h2>
+        <p>Use the checkbox parameter to get an HTML5 number field. Checkbox does not show the changed indicator.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field checkbox [model]="model" name="complete" prompt="Are you done yet?" label="Foo"&gt;
+            &lt;div class="fancy-hint"&gt;psst, you're not.&lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field checkbox [model]="model" name="complete" prompt="Are you done yet?" label="Foo">
+          <div class="fancy-hint">psst, you're not.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>select</code></h2>
+        <p>
+          Use the select parameter to get a select dropdown.
+          Options can be provided as a single array of strings or an array of arrays of <code>[display: string, value: any]</code>.
+        </p>
+        Usage:
+        <pre class="code">
+          const colors = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'purple'];
+          // OR 
+          const colors = [['red', 'R'], ['yellow', 'Y'], ['blue', 'B']]
+          
+          &lt;prx-fancy-field [model]="model" name="color" label="Favorite color" [select]="colors"&gt;
+            &lt;div class="fancy-hint"&gt;It doesn't have to be your absolute favorite, but you can only pick one.&lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field [model]="model" name="color" label="Favorite color" [select]="colors">
+          <div class="fancy-hint">It doesn't have to be your absolute favorite, but you can only pick one.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>textarea</code></h2>
+        <p>Use textarea for longer text fields.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field textarea [model]="model" name="script" label="Your script"&gt;
+            &lt;div class="fancy-hint"&gt;
+              This sponsor does not require final script approval.
+              Save your script here for storage and convenience.
+            &lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field textarea [model]="model" name="script" label="Your script">
+          <div class="fancy-hint">
+            This sponsor does not require final script approval.
+            Save your script here for storage and convenience.
+          </div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>.nested</code></h2>
+        <p>Use content projection with the <code>.nested</code> selector to include anything in a fancy-field. Maybe a date picker?</p>
```

Comment:
are we doing anything with the .nested selector here in this example? or maybe i'm not 150% clear on how it should be used...